Open_Source_Events/MWOC/README.md

```diff
@@ -0,0 +1,15 @@
+<h1 align="center"> MEXILI WINTER OF CODE (MWOC) </h1>
+
+## About MWOC
+
+Mexili is a Free and Open Source organization whose motivation is to funnel learning and implementation through Open Source Software Development.
+
+Mexili is an organization that strives towards building a community of enthusiastic and passionate individuals and developers who want to work on building open source projects and learn real software development skills while also targeting to make projects that have a positive impact in the society and help improve productivity alike.
+
+Whether you have an unfinished hackathon project or a completely new idea for an open-source project, at Mexili you can grow and nurture it into a full-fledged product and make an impact towards the community. With an enthusiastic community of developers, Mexili thrives on collaboration and a thriving community of helpful and skilled developers.
```

Comment:
Can you add more about MWoC instead of Mexili itself? 

Mexili Winter of Code has a nice description about their OSS Program here: https://mexili.github.io/winter_of_code/#/winter_of_code 

Also, I would appreciate that instead of Copy-Pasting stuff, you can come up with your own words, run it against a Grammarly Check and then push your commits.

Reply:
Yes, I will do that. I missed the about program texts in the commit. I had a look at 1 or 2 other such READMEs in the same directory and there the content was copy-pasted from the program's website so I thought that this is how it is supposed to done. I will immediately start working and change it to some original text, sorry for that.

Reply:
You don't have to be sorry. Just a heads up because I am doing tight-checks on every PR these days